lib/bitbucket_rest_api/repos.rb

```diff
@@ -63,7 +64,9 @@ def sources
     def services
       @services ||= ApiFactory.new 'Repos::Services'
     end
-
+    def forks
+      @services ||= ApiFactory.new 'Repos::Forks'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.